lib/datahub-client/data_platform_catalogue/entities.py

```diff
@@ -379,6 +392,7 @@ class Database(Entity):
         description="Unique identifier for the entity. Relates to Datahub's urn",
         examples=["urn:li:container:my_database"],
     )
+    database_entities: list = Field(description="list of tables in the database")
```

Comment:
Think we can shorten this to `entities` or `tables` now, since the database part is obvious from the class.